flink-end-to-end-tests/flink-end-to-end-tests-common/pom.xml

```diff
@@ -61,7 +61,7 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-dist_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
-			<scope>provided</scope>
+			<scope>compile</scope>
```

Comment:
hmm....why not provided?

Reply:
In that case we should add a dependency to flink-runtime but keep flink-dist provided; flink-dist pulls in a loooot of unnecessary stuff.

Reply:
Well, checked that stacktrace again.  it's not a compiling exception but a runtime exception, so actually, the flink_dist can be set provided ( no need to be compile any more).  Sorry about the noise.